### PR TITLE
 connecting API 

### PR DESCRIPTION
**Pull request by massazoum**
In this project,

- I connected API via redux.
- I used the data to display something in a UI.

### CODE REVIEW

File: src/redux/books/BookSlice.js [hunk 1]
```diff
@@ -1,36 +1,54 @@
...
+export const postData = createAsyncThunk('data/postData', async () => {
+  try {
+    const response = await axios.post('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/');
+    console.log(response.data);
+  } catch (error) {
+    console.log(error);
+  }
+});
+
+export const getListBook = createAsyncThunk('data/getListBook', async () => {
```
Comment (joskalenda) [inline src/redux/books/BookSlice.js]: - [ ] The image's requirement said it all clearly, please create an action called fetchbooks, I think it does matter to follow the need and update the action name everywhere in the app

File: src/redux/books/BookSlice.js [hunk 2]
```diff
@@ -1,36 +1,54 @@
...
+        'Content-Type': 'application/json',
+      },
+    });
+    console.log(response);
+  } catch (error) {
+    console.log(error);
+  }
+});
+
+export const deletebook = createAsyncThunk('data/deletebook', async (IDD) => {
```
Comment (joskalenda) [inline src/redux/books/BookSlice.js:36]: - [ ] As for these 2 actions, when they are called, the state is not updated unless I manually refresh the page, I mean when a new book is added or a book is deleted, you need to manually reload the page for the changes to apply which is not a good idea at all, please fix it

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @massazoum , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Green inters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by divinecharlotte — APPROVED** [review]
# Hello @massazoum     😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯

- Descriptive pr👍 
- Linters are passing 👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**